contrib/ruby/lib/trilogy.rb

```diff
@@ -7,7 +7,18 @@
 require "trilogy/encoding"
 
 class Trilogy
+  VALID_OPTIONS = %i[
+    host port path database username password encoding
```

Comment:
What is the distinction between `socket` and `path` options? 

I see this:

https://github.com/trilogy-libraries/trilogy/blob/362ee5881a0a92867895d5bdfeb83165316bddc4/contrib/ruby/script/benchmark#L15-L16

..but also this:

https://github.com/trilogy-libraries/trilogy/blob/362ee5881a0a92867895d5bdfeb83165316bddc4/contrib/ruby/test/test_helper.rb#L61

Reply:
I don't think `path` is a valid option for either trilogy or mysql. Maybe it used to be called `path`?